README.md

```diff
@@ -67,6 +67,18 @@ scraper to test out config changes.
   * Generate documentation for all the modules in your PDK (where possible) and
     put in a folder inside your version docs
 
+## Generating the Admin API Documentation
+
+- Have a local clone of Kong
+- Install resty-cli (comes with Kong)
+- In the Kong repository, checkout the desired branch/tag/release
+- Run: `KONG_PATH=path/to/your/kong/folder KONG_VERSION=0.14.x gulp admin-api-docs`
```

Comment:
We could make so that, given a full version tag `X.Y.Z`, the gulp taks could checkout a local clone of the tag in a temp dir (and make this temp dir `KONG_APTH`), then derive the `X.Y.x` name for `KONG_VERSION` based on the tag. This would make this process more robust: there wouldn't be a chance that someone would point to the wrong version/checkout and produce docs matching a wrong version.

Reply:
For the future us :)